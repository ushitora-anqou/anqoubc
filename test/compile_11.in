-1 + 1;
1 + -1;
4 / -2;
4 * -2;
-1. + 1.;
1. + -1.;
4. / -2.;
4. * -2.;
5. / 2;
