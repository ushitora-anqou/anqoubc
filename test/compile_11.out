0i
0i
-2i
-8i
0.000000f
0.000000f
-2.000000f
-8.000000f
2.500000f
